jaxley/modules/compartment.py

```diff
@@ -93,3 +93,6 @@ def _init_morph_jax_spsolve(self):
         self._data_inds = data_inds
         self._indices_jax_spsolve = indices
         self._indptr_jax_spsolve = indptr
+
+        # To enable updating `self._comp_edges` during `View`.
+        self._comp_edges_in_view = self._comp_edges.index.to_numpy()
```

Comment:
any reason why this cannot be part of Module and needs to be part of Compartment, Branch, Cell, Network?

Reply:
and is there a reason why comp_edges and comp_edges_in_view have different types? array vs DataFrame?

Reply:
1) Because it depends on `comp_edges`, which is specific to each type of Module.
2) It's the same for `nodes` and `nodes_in_view` as well as `edges` and `edges_in_view`.